lib/ljglibs/gio/file.moon

```diff
@@ -57,6 +66,17 @@ core.define 'GFile', {
   enumerate_children: (attributes, flags = @QUERY_INFO_NONE) =>
     gc_ptr catch_error C.g_file_enumerate_children, @, attributes, flags, nil
 
+  copy: (dest, flags, cancellable, callback) =>
```

Comment:
Here, for the for parsing the flags I'd like to see [core.parse_flags](https://github.com/howl-editor/howl/blob/master/lib/ljglibs/core.moon#L163) being used. We should actually update `query_info` above to use the same thing, I think this file predates this helper. With this in place we could remove the flags handling from howl's `File`. The code would look like this:

```moonscript
flags = core.parse_flags('G_FILE_', opts.flags)
```

`callback` is too generic a name, `progress_callback` provides an idea of what it is.